src/Utility/Text.php

```diff
@@ -801,7 +801,7 @@ protected static function _substr(string $text, int $start, ?int $length, array
             $len = self::_strlen($part, $options);
             if ($offset !== 0 || $totalLength + $len > $length) {
                 if (
-                    strpos($part, '&') === 0
+                    mb_strpos($part, '&') === 0
```

Comment:
Should the preg_ functions be multi-byte or unicode?

Reply:
The pattern on 785 is looking for HTML entities and hexadecimal entities. Those patterns shouldn't require unicode support.

Reply:
IDK, I just changed this one as other functions in the method where `mb_*`.